sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/CreateRsaKeyOptions.java

```diff
@@ -30,107 +29,150 @@
     private int publicExponent;
 
     /**
-     * Creates a RsaKeyCreateOptions with {@code name} as name of the Rsa key.
+     * Creates a RsaKeyCreateOptions with {@code name} as name of the RSA key.
+     *
      * @param name The name of the key.
      */
     public CreateRsaKeyOptions(String name) {
         super(name, KeyType.RSA);
     }
 
     /**
-     * Get the keySize value.
+     * Get the key size.
```

Comment:
```suggestion
     * Get the key size in bits.
``` 

Please update in other places as well.